### PR TITLE
chore: add pull request template

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
# Add pull request template

## Description
Adds a pull request template, based on [DAB template](https://github.com/ansible/django-ansible-base/blob/devel/.github/PULL_REQUEST_TEMPLATE.md?plain=1), to standardize the pull request format. 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Test update
- [ ] Refactoring (no functional changes)
- [ ] Development environment change
- [ ] Configuration change
- [X] CI change

## Self-Review Checklist

- [X] I have performed a self-review of my code
- [ ] I have added relevant comments to complex code sections
- [ ] I have updated documentation where needed
- [ ] I have considered the security impact of these changes
- [ ] I have considered performance implications
- [ ] I have thought about error handling and edge cases
- [ ] I have tested the changes in my local environment
- [ ] I have run the linters and test suite locally
- [ ] I have tested the changes on integration environment

## Testing Instructions
Not applicable

### Expected Results
Not applicable

## Additional Context
Not applicable

### Required Actions
Not applicable

### Screenshots/Logs
Not applicable

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# [AAP-XXXXX] Brief Title Describing the Change
```
Comment (kaiokmo) [inline .github/pull_request_template.md:1]: should the type of change prefix (e.g.: `feat`, `fix`, `chore`) be specified in the PR title as well, or we are deprecating it in favor of `"## Type of Change"` just to be informative? does it have any impact on our release/semantic versioning processes or github actions?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+## Description
+<!-- Mandatory: Provide a clear, concise description of the changes and their purpose -->
+- What is being changed?
+- Why is this change needed?
+- How does this change address the issue?
+
+Closes #XXX (if applicable)
+Jira: [AAP-XXXXX](https://jira.aap.com/browse/AAP-XXXXX) (if applicable)
+
+## Type of Change
```
Comment (kaiokmo) [inline .github/pull_request_template.md:12]: ditto title

File: .github/pull_request_template.md (hunk 1 above)
Comment (Alex-Izquierdo) [reply .github/pull_request_template.md:1]: We are following that pattern but it is not enforced or stated at any place, so for now I understand is optional.

File: .github/pull_request_template.md (hunk 2 above)
Comment (Alex-Izquierdo) [reply .github/pull_request_template.md:12]: Sorry, I do not follow you. What do you mean?

File: .github/pull_request_template.md (hunk 1 above)
Comment (kaiokmo) [reply .github/pull_request_template.md:1]: I see. So, in this case, having `feat: [AAP-XXXXX]` or `fix: [AAP-XXXXX]` has no implications and would be optional.

**Review by kaiokmo — APPROVED** [review]

**Review by zkayyali812 — APPROVED** [review]
